README.md

```diff
@@ -1,94 +1,107 @@
-* [ColorLib](https://github.com/c0rp3n/colorlib-sm/tree/master/addons/sourcemod/scripting/include)
+**Pull requests and any contributions are welcome and encouraged!**
+**Our [discord](https://discord.gg/5dsFHeR) is a good place for dev talk and help!**
 
 ## Issue Rules
 
+**We are only helping with timer related bugs, any host related issue will be closed immediately.**
+
 **If any of the rules listed below are not followed, you must expect the issue to be closed immediately.**
 
-- Requirements:
-	- Ensure your timer version is up to date with the latest release
-	- SourceMod and Metamod are up to date (support will only be given for latest stable versions)
-	- Ensure includes for compilation are up to date
-	- Using the stock timer without any additional changes
-- You're following the template
+- Prerequisites:
+	- Ensure your timer version is up to date with the [latest release](https://github.com/olokos/Surftimer-olokos/releases/latest)
+	- [SourceMod](https://www.sourcemod.net/downloads.php?branch=stable) and [Metamod](https://www.sourcemm.net/downloads.php/?branch=stable) are up to date (support only for latest stable)
+	- Ensure [includes](https://github.com/olokos/Surftimer-olokos#Optional) for compilation are up to date
+	- Stock timer without any additional changes
+- Follow the template
 	- That means you won't delete any pre-entered questions!
-- You're giving clear information
-- You won't edit issues - you always write a new comment below!
-- **Any community/server specific bug/suggestion will be ignored/closed!**
+- Give clear and as precise information as you can
+	- If applicable include related logs / configs (posted to [pastebin](https://pastebin.com/))
+- Don't edit issues 
+	- Always write a new comment below!
+	
+	
+## Installation
 
-## Fresh Install
+#### Requirements
 
-*   Clone or download the repository ([Link](https://github.com/olokos/Surftimer-olokos-public/archive/master.zip))
-*   Obtain all of the [compilation requirements](https://github.com/olokos/Surftimer-olokos#installation-and-requirements)
-*   Download latest stable SourceMod version (1.10+) for your OS ([Link](https://www.sourcemod.net/downloads.php?branch=stable))
-*   Windows: Put spcomp.exe and compile.exe in scripting folder and double click compile.exe
-*   If there are no errors, (warnings are fine, for now) move .smx files from compiled to /plugins
-*   Copy the rest of the files from this repository to your csgo directory
-*   Edit configs (databases.cfg, admins, etc.)
+* [SourceMod 1.10](https://www.sourcemod.net/downloads.php?branch=stable)
+* [Metamod 1.10](https://www.sourcemm.net/downloads.php/?branch=stable)
+* [DHooks](https://forums.alliedmods.net/showpost.php?p=2588686&postcount=589)
+* [SMJansson](https://forums.alliedmods.net/showthread.php?t=184604)
+* [SteamWorks](https://forums.alliedmods.net/showthread.php?t=229556)
+* [Sourcemod-Discord API](https://github.com/Deathknife/sourcemod-discord)
```

Comment:
Since we provide the smx file... is it necessary to keep this as Requirement?

Reply:
Yes this is required, without this surftimer wont compile and people will have difficulties building dev on their own.

We're planning to make it fully modular, but for now this should remain.
Correct me if I'm wrong though.